core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -977,6 +977,294 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .build();
 
+  // Rocks Block metrics
+  public static final MetricKey MASTER_ROCKS_BLOCK_BACKGROUND_ERRORS =
+      new Builder("Master.RocksBlockBackgroundErrors")
+          .setDescription("RocksDB block table. Accumulated number of background errors.")
```

Comment:
If this metric is non-zero, what should the user do? Check rocksDB error log somewhere?

Reply:
It's for informational purposes, but we can remove it if you think it will cause confusion.